Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStore.cs

```diff
@@ -223,14 +228,16 @@ private CentralStorage CreateCentralStorage(CentralStoreConfiguration configurat
         }
 
         /// <nodoc />
-        public void PreStartupInitialize(Context context, ILocalContentStore localStore, IDistributedContentCopier copier)
+        public void PreStartupInitialize(Context context, ILocalContentStore localStore, IDistributedContentCopier copier, Func<OperationContext, ContentHash, Task<ResultBase>> proactiveCopyTaskFactory)

```

Comment:
>Func<OperationContext, ContentHash, Task<ResultBase>> proactiveCopyTaskFactory [](start = 123, length = 78)

This seems a little strange. Likely, it is a result of doing proactive replication inside of LLS. I would think that it should instead occur inside of the DistributedContentStore.